src/index.scss

```diff
@@ -5,6 +5,11 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 
 @link https://github.com/carbon-design-system/carbon/issues/2789
 */
+
+@import 'carbon-components/scss/globals/scss/styles.scss';
 main {
   display: block;
 }
+$feature-flags: (
```

Comment:
The feature-flag mapping should come before the styles import.  See https://www.carbondesignsystem.com/tutorial/react/step-1#add-ui-shell-sass